libfreerdp/core/tcp.c

```diff
@@ -1163,10 +1162,16 @@ int freerdp_tcp_default_connect(rdpContext* context, rdpSettings* settings, cons
 #if defined(HAVE_AF_VSOCK_H)
 		hostname = vsock;
 		sockfd = socket(AF_VSOCK, SOCK_STREAM, 0);
+		if (sockfd < 0)
```

Comment:
I think a warning message with the `errno` would be good here (when hitting that issue).

Reply:
ok, done.